mhs/common/data/interactions/interactions.json

```diff
@@ -12,7 +12,7 @@
     "ack_requested": false,
     "ack_soap_actor": "urn:oasis:names:tc:ebxml-msg:actor:toPartyMSH",
     "sync_reply": true,
-    "sync-async": false
+    "sync_async": true
```

Comment:
As I understand this is actually what it will be, interactions will be listed as whether or not they can be wrapped as sync async then the supplier will have the choice in the header